assignment_3/analysis.py

```diff
@@ -263,9 +288,16 @@ def plot_heatmap(data):
 chain_length_data = pd.read_csv("./data/chain_length_results.csv", header=0)
 train_data = pd.read_csv("./data/train_results.csv")
 
+
+small_map_data = pd.read_csv("./method_results_small.csv", header=0)
```

Comment:
./data/method_results_small.csv since i moved all csv under data folder. please update the dataset if you have the latest one